notebooker/web/templates/result_listing.html

```diff
@@ -56,5 +59,14 @@ <h1 class="ui huge centered header">{{ titleised_report_name }} ({{ report_name
             <div class="ui cancel button">Cancel</div>
           </div>
         </div>
+<script type="application/javascript">
+    document.getElementById('deleteAllReportsButton').addEventListener('click', function(e) {
+        e.preventDefault();
+        var confirmed = confirm('Are you sure you wish to delete all reports?');
+        if (confirmed) {
+            window.location.href = this.getAttribute('data-href');
```

Comment:
This could take a long time - do we want to display a spinner or even a status update to the user?